### PR TITLE
janitoring: Better readability

### PR DESCRIPTION
**Pull request by vargac**
- Added comments
- "Nothing-saying" numbers changed to constants

### CODE REVIEW

File: PixyViSy.cpp [hunk 1]
```diff
@@ -15,48 +15,77 @@ void PixyViSy::update(void)
...
+              max[2] = max[1];
+              max[1] = max[0];
+              max[0] = i;
+            }
+            else {
+              max[2] = max[1];
+              max[0] = i;
+            }
+          }
+          else {
```
Comment (mrshu) [inline PixyViSy.cpp]: @vargac I think we should decide on which way of formatting will be used in this project. I personally like the other way (having `}` on the same line as `else {`) a bit more.

File: PixyViSy.cpp [hunk 2]
```diff
@@ -15,48 +15,77 @@ void PixyViSy::update(void)
   uint8_t all_blocks_count = pixy.getBlocks();
   left_pixels = 0;
   right_pixels = 0;
-  action = 'N';
+  action = 'N'; // as "Nothing"
   distance = ~0;
   uint16_t goal_pix_height = 0;
+  uint8_t max[3] = { all_blocks_count, all_blocks_count, all_blocks_count };
```
Comment (mrshu) [inline PixyViSy.cpp]: @vargac if `all_blocks_count` is the number of blocks, will `pixy.blocks[max[0]]` for instance be defined? I do not think so.

**Comment by mrshu** [comment]
@vargac very nicely done!

Apart from that one comment I think it is ready to get merged in.

File: PixyViSy.h [hunk 3]
```diff
@@ -5,6 +5,11 @@
 #include <Pixy.h>
 #include <stdint.h>
 
+#define MIN_BLOCK_AREA 100
```
Comment (mrshu) [inline PixyViSy.h:8]: @vargac just out of curiosity, why 100?

File: PixyViSy.cpp (hunk 1 above)
Comment (vargac) [inline PixyViSy.cpp]: @mrshu ok, I can change it

File: PixyViSy.h (hunk 3 above)
Comment (vargac) [inline PixyViSy.h:8]: @mrshu Just guess. I thought it will be changed somewhen in the future.

**Comment by mrshu** [comment]
@vargac Nicely done, this looks pretty good now!

**Pull request merged**